packages/block-library/src/button/edit.js

```diff
@@ -64,86 +93,100 @@ class ButtonEdit extends Component {
 			fallbackBackgroundColor,
 			fallbackTextColor,
 			setAttributes,
-			isSelected,
 			className,
+			instanceId,
 		} = this.props;
 
 		const {
 			text,
 			url,
 			title,
+			linkTarget,
+			rel,
 		} = attributes;
 
+		const id = `wp-block-button__inline-link-${ instanceId }`;
```

Comment:
Nitpick - the variable name could be slightly more specific - i.e. `linkId`.

Reply:
Thank you! Addressed in [this commit](https://github.com/WordPress/gutenberg/pull/10128/commits/cb7242cabbc56b3511dc7cb06a141d4bb76e31c0) ❤️